pkg/cluster/internal/providers/docker/provision.go

```diff
@@ -271,18 +283,20 @@ func runArgsForLoadBalancer(cfg *config.Cluster, name string, args []string) ([]
 	return append(args, loadbalancer.Image), nil
 }
 
-func getProxyEnv(cfg *config.Cluster) (map[string]string, error) {
+func getProxyEnv(cfg *config.Cluster, networkName string, nodeNames []string) (map[string]string, error) {
```

Comment:
should `[]nodeNames` be in `common` and not provider specific?

Reply:
nah, there's no DNS in the others at the moment. we can DRY out later if that becomes the case